geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/pubsub/PubSubImpl.java

```diff
@@ -66,16 +69,18 @@ public int getSubscriptionCount() {
   public long publish(Region<RedisKey, RedisData> dataRegion, byte[] channel, byte[] message) {
     PartitionRegionInfo info = PartitionRegionHelper.getPartitionRegionInfo(dataRegion);
     Set<DistributedMember> membersWithDataRegion = new HashSet<>();
-    for (PartitionMemberInfo memberInfo : info.getPartitionMemberInfo()) {
-      membersWithDataRegion.add(memberInfo.getDistributedMember());
+    if (info != null) {
```

Comment:
Is there a test that checks the necessity of this code?

Reply:
Looking into it a bit, this null check should be unnecessary, as `PartitionRegionHelper.getPartitionRegionInfo()` only returns null if we hit a `ClassCastException` during it, but nothing in that method actually throws one. It's outside the scope of these changes to go fixing it though, so for now I'll just remove the check and let IntelliJ complain about it.